src/views/Tests.jsx

```diff
@@ -47,19 +47,20 @@ const Tests = ({ setTests, tests }) => {
                     </thead>
                     <tbody>
                       {
-                        tests.map(test => (
+                        tests.map(testsuite => (
+                            testsuite.tests.map(test =>
                         <tr key={test.id}>
                           <td>{test.id}</td>
-                          <td>SEC0001</td>
-                          <td>{test.url}</td>
+                          <td>{test.code}</td>
+                          <td>{testsuite.url}</td>
                           <td className="text-danger">
                             <i className="tim-icons icon-alert-circle-exc"/>
-                            FAILED
```

Comment:
there should be probably some conditional about which icon should be rendered depending on test result